### ISSUE TITLE
Leaving the status buffer modifies the active theme colors of all buffers

### ISSUE THREAD

**Issue by mfiano** [TimUntersberger/neogit#60]
When I'm done with the neogit status buffer, exiting it results in my theme changed. I was unable to figure out how to revert it without restarting Neovim. It seems to mutate the fold color of the onedark theme.

Before: ![](https://i.lisp.cl/EdkQiQ.png)
After: ![](https://i.lisp.cl/NXnk5E.png)

**Comment by RianFuro** [comment]
Found a better way :) @mfiano can you double check? I'm not using a colorscheme that styles folds so I'm not completely sure it's fixed.

**Comment by mfiano** [comment]
Well try now, thanks. I also found another neogit bug regarding fold theming, and I'll see if it fixes that before I report it.

**Comment by mfiano** [comment]
@RianFuro The original issue seems to have been solved. The fold color issue I recently discovered is still a problem though, and is specific to the neogit status buffer. Should I make a new issue for that?

**Comment by RianFuro** [comment]
Yes, please. Closing here then :)
